bokeh/embed.py

```diff
@@ -528,6 +582,17 @@ def autoload_server(model=None, app_path=None, session_id=None, url="default", r
             when running the Bokeh behind reverse-proxies under certain
             configurations
 
+        resources (str) : A string specifying what resources need to be loaded
+            along with the document.
+
+            If ``default`` then the default JS/CSS bokeh files will be loaded.
+
+            If None then none of the resource files will be loaded. This is
+            useful if you prefer to serve those resource files via other means
+            (e.g. from a caching server). Be careful, however, that the resource
+            files you'll load separately are from the same version as the
+            server, otherwise the rendering may not work correctly.
+
```

Comment:
It looks like it isn't actually added to the params, just the docstring entry should be removed.